index.md

```diff
@@ -16,98 +16,91 @@ order: 1
 
 ## Features of Evaluation Tools
 {:#Features}
 
-Web accessibility evaluation tools target different audiences. This includes designers, developers, non-technical content authors, quality assurance testers, and sometimes also end-users. Tools tend to offer different features and functionality which may help users to compare and assess Web accessibility evaluation tools for their specific needs.
+Web accessibility evaluation tools target different audiences. This includes designers, developers, non-technical content authors, quality assurance testers, and sometimes also end-users. Tools tend to offer different features and functionality which may help users to compare and assess web accessibility evaluation tools for their specific needs.
 
-The following features describe tools found on the [Web Accessibility Tool List](https://www.w3.org/WAI/ER/tools/). 
-
-### Features You Can Use as a Filter
-
-You may use the following features to reduce the number of evaluation tools listed.
+The following describes tool features found on the [Web Accessibility Evaluation Tools List](https://www.w3.org/WAI/ER/tools/). 
 
 * **Guidelines**  
 Different organizations and governments may require conformance with different accessibility standards, thus different tools support these standards. The W3C Web Content Accessibility Guidelines (WCAG) is internationally recognized as the standard for web accessibility, and supported by most tools.
 
 * **Language**  
-Evaluation tools support different languages. This includes the user interface of the tools, as well as the language of the content they support (for example for detecting language-related barriers).
+The support for different languages provided by evaluation tools. This includes the user interface of the tools, as well as the language of the content they support (for example for detecting language-related barriers).
 
 * **Type of Tool**  
-Lists the platform and type of tools that are available, e.g. authoring tool or browser plugin, command line, desktop or mobile application.
+Evaluation tools can be plugins (extensions) for authoring tools and web browsers, command line tools, desktop or mobile applications, and online services. Diffent tools may be combined, depending on your design and development process.
 
-* **Supported Formats**  (referred to as "Technology" in the filters)
-Lists the format the tool can test, e.g. HTML, CSS, PDF, etc.
+* **Supported Formats**
+Most evaluation tools check the accessibility of HTML content. Some also support other web technologies, such as WAI-ARIA, CSS, SVG, and PDF.
 
 * **Assists by**  
-Lists what the specific uses of the tool, e.g. generating reports, providing step-by-step guidance, or displaying information within a web page
```

Comment:
This reflects exactly what the evaluation tools page uses for the text. The changes proposed here step away from this list of features as outlined on the tools page.

In fact, all the reorganization of the page's content have an impact on the work @lakeen have been doing.

Reply:
But these *are* filterable features, so your organization seems really confusing to me. Why separate a feature from its sub-features?